src/aat/resources/features/F-027/S-035.td.json

```diff
@@ -14,13 +14,13 @@
   "expectedResponse": {
     "_extends_": "Common_400_Response",
     "headers" : {
-      "Content-Length" : "[[ANY]]",
+      "Content-Length" : "[[ANYTHING_PRESENT]]",
```

Comment:
ANY_INTEGER_NULLABLE can be used here.. This applies to all the Content-Length headers